src/poliastro/plotting.py

```diff
@@ -399,6 +327,95 @@ def savefig(self, filename, **layout_kwargs):
         )
 
 
+class OrbitPlotter3D(_BaseOrbitPlotter):
+    """OrbitPlotter3D class.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._layout = Layout(
+            autosize=True,
+            scene=dict(
+                xaxis=dict(
+                    title="x (km)",
+                ),
+                yaxis=dict(
+                    title="y (km)",
+                ),
+                zaxis=dict(
+                    title="z (km)",
+                ),
+                aspectmode="data",  # Important!
+            ),
+        )
+
+    @u.quantity_input(elev=u.rad, azim=u.rad, distance=u.km)
+    def set_view(self, elev, azim, distance=5 * u.km):
+        x = distance * np.cos(elev) * np.cos(azim)
+        y = distance * np.cos(elev) * np.sin(azim)
+        z = distance * np.sin(elev)
+
+        self._layout.update({
+            "scene": {
+                "camera": {
+                    "eye": {
+                        "x": x.to(u.km).value, "y": y.to(u.km).value, "z": z.to(u.km).value
+                    }
+                }
+            }
+        })
+
+    def _redraw_attractor(self, min_radius=0 * u.km):
+        # Select a sensible value for the radius: realistic for low orbits,
+        # visible for high and very high orbits
+        radius = max(self._attractor.R.to(u.km), min_radius.to(u.km))
+        if radius < self._attractor_radius:
+            # If the resulting radius is smaller than the current one, redraw it
+            shape = _plot_sphere(radius, BODY_COLORS.get(self._attractor.name, "#999999"), self._attractor.name)
+            # Overwrite stored properties
+            self._attractor_radius = radius
+            self._attractor_data = shape
+
+    def _plot_trajectory(self, trajectory, label, color, dashed):
+        trace = Scatter3d(
+            x=trajectory.x.to(u.km).value, y=trajectory.y.to(u.km).value, z=trajectory.z.to(u.km).value,
+            name=label,
+            line=dict(
+                color=color,
+                width=5,
+                dash='dash' if dashed else 'solid',
+            ),
+            mode="lines",  # Boilerplate
+        )
+        self._data.append(trace)
```

Comment:
What about moving `self._data.append(trace)` to the base class and call `super()` at the end of `_plot_trajectory` for both subclasses?

Reply:
Apart from this, resolved everything! Is this necessary? As it would anyways increase number of lines

Reply:
Nah, let's leave it like this.